src/client/helpers/utils.tsx

```diff
@@ -161,3 +163,17 @@ export function dateObjectToISOString(value: DateObject) {
 	}
 	return date;
 }
+
+export function stringToUrl(string: string) {
```

Comment:
I think the name could be a bit more descriptive, maybe something along those lines ?
```suggestion
export function stringToHTMLWithLinks(string: string) {
```